src/disk_io_thread.cpp

```diff
@@ -1087,10 +1104,6 @@ namespace libtorrent
 		TORRENT_ASSERT(j->next == 0);
 		TORRENT_ASSERT((j->flags & disk_io_job::in_progress) || !j->storage);
 
-		mutex::scoped_lock l(m_cache_mutex);
-
-		check_cache_level(l, completed_jobs);
-
 		DLOG("perform_job job: %s ( %s%s) piece: %d offset: %d outstanding: %d\n"
```

Comment:
when I moved this in my other patch, I assumed this log statement required the cache_mutex to be held. I'm not actually sure that's the case though. but this patch moves it out from under that lock